frontend/javascripts/components/color_picker.tsx

```diff
@@ -68,9 +85,99 @@ export function ChangeColorMenuItemContent({
   const content = isDisabled ? null : (
     <ThrottledColorPicker color={color} onChange={onChangeColor} />
   );
+  return getPopover(title, content);
+}
+
+const ThrottledRGBAColorPicker = ({
+  color,
+  onChangeColor,
+}: {
+  color: RgbaColor;
+  onChangeColor: (color: RgbaColor) => void;
+}) => {
+  const [value, localSetValue] = useState(color);
+  const throttledSetValue = useThrottledCallback(onChangeColor, [onChangeColor, value], 20);
+  const setValue = (newValue: RgbaColor) => {
+    localSetValue(newValue);
+    throttledSetValue(newValue);
+  };
+  const setValueFromHex = (color: string) => {
+    const colorRgb = Utils.hexToRgb(color);
+    setValue({ r: colorRgb[0], g: colorRgb[1], b: colorRgb[2], a: value.a });
+  };
+
+  const maybeGetInfoText = () => {
+    return (
+      <div style={{ wordBreak: "break-word", fontSize: "12px", lineHeight: 1, marginTop: "8px" }}>
```

Comment:
```suggestion
      <div style={{ wordBreak: "break-word", fontSize: 12, lineHeight: 1, marginTop: 8 }}>
```

Reply:
please look for other "px" values in this module, too. i find it easier to maintain when quotes and "px" are omitted.